api/src/backend/tasks/tasks.py

```diff
@@ -414,3 +444,102 @@ def check_lighthouse_connection_task(lighthouse_config_id: str, tenant_id: str =
             - 'available_models' (list): List of available models if connection is successful.
     """
     return check_lighthouse_connection(lighthouse_config_id=lighthouse_config_id)
+
+
+@shared_task(
+    name="integration-check",
+    queue="integrations",
+)
+def check_integrations_task(
```

Comment:
I think something like `run_integrations` would be clearer, what do you think? At first I thought this was just checking the integration status.